x-pack/plugins/cases/public/components/files/files_utility_bar.test.tsx

```diff
@@ -16,6 +16,7 @@ import { FilesUtilityBar } from './files_utility_bar';
 const defaultProps = {
   caseId: 'foobar',
   onSearch: jest.fn(),
+  onSearchType: jest.fn(),
 };
 
 describe('FilesUtilityBar', () => {
```

Comment:
Could you please:

1. Update the `renders correctly` test to `expect(await screen.findByTestId(` the test Id of TypeFilterPanel `)).toBeInTheDocument();`
2. Add a new test for 'type filter passed correctly to callback' to make sure changing the select component calls `onSearchType` with the correct `string[]`.